source/installation-guide/more-installation-alternatives/wazuh-from-sources/wazuh-server/index.rst

```diff
@@ -51,7 +51,7 @@ Installing Wazuh manager
           .. code-block:: console
 
             # yum install make cmake gcc gcc-c++ python3 python3-policycoreutils automake autoconf libtool openssl-devel cmake
-            # rpm -i $(rpm --eval https://packages.wazuh.com/utils/libstdc%2B%2B/libstdc%2B%2B-static-8.4.1-1.el8.'%{_arch}'.rpm)
+            # rpm -i $(rpm -q libstdc++-devel --queryformat "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libstdc++-static-%{VERSION}-%{RELEASE}.%{arch}.rpm")
```

Comment:
Modify to 

```suggestion
            # rpm -i $(rpm -q libstdc++-devel --queryformat "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libstdc++-static-%{VERSION}-%{RELEASE}.%{arch}.rpm\n")
```
this is since an environment can have multiple libstdc++ versions installed (suppose you have two architectures installed), with your code this can cause a concatenation in the output of **rpm -q** command, which would lead to a 404 error.

Reply:
Done!